text/XXXX-delete_and_drop.md

```diff
@@ -0,0 +1,225 @@
+- RFC PR: [#53](https://github.com/qri-io/rfcs/pulls/53)
+- Issue: <!-- (leave this empty) -->
+
+# Summary
+[summary]: #summary
+
+Rename the remove command to `delete`, and clarify it's purpose is for deleting versions from a dataset history. Create a new command `drop` for removing datasets from remotes.
+
+# Motivation
+[motivation]: #motivation
+
+The _user experience (UX)_ of removing data from Qri needs some work. We're seeing a number of users get into states they feel they cannot recover from, and rather than use tools like `qri remove`, they're deleting repositories entirely in an effort to start over. 
+
+A better UX for removing data from qri should clearly answer two common "uh oh" scenarios a user is likely to encounter:
+
+1. "I made a mistake while saving"
+2. "I want to undo a push"
+
+The first is an example of removing data from history. The second is an example of removing data from the network. In both scenarios the user is a data author, and a good answer to these questions should present well-documented, easy-to-find commands that when executed leave the user feeling that they're in a clean state. 
+
+A well-documented desctrive command should be clearly labeled as the "opposite" of a constructive command and present warnings _before_ execution. Once executed, commands remove data should show as few caveat/errors/warnings as possible, or at least connect caveats to a clear next course of action. A feeling of "clean state" comes when vommands like `qri log`, `qri list` and `qri status` match user expectations after running a destructive command.
+
+To improve our "undo UX", this RFC proposes three changes to qri CLI:
+1. rename `remove` to `delete`, clarify its text to talk about history instead of just storage
+2. Add an `amend` flag on the `save` command that _replaces_ the HEAD commit
+3. Add a new top-level command: `drop`, for dropping datasets you don't own
+
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+### Renaming Remove
+Here's the current (qri v0.9.8) description of the `remove` command:
+
+```
+$ qri remove --help
+Remove gets rid of a dataset from your qri node. After running remove, qri will
+no longer list your dataset as being available locally. 
+[rambling bit about IPFS storage redacted]
+```
+
+This text is misleading. It seems to be about storage, when the primary purpose of  `remove` is to edit history. Storage savings are a byproduct of editing one or more commits out of a history. This RFC sets aside the tight coupling between history and storage for another day, and instead focueses on fixing the role of remove by clearifying it's documentation:
+
+```
+$ qri delete --help
+Delete removes commits from a dataset history. Use delete to "correct the record", 
+removing commits in a dataset history. Delete always works from the latest (HEAD) 
+commit, working backwards toward the first commit. To delete from a dataset you 
+must be able to edit it.
```

Comment:
Generally, I'm very happy with this change; remove was not clear that it was only editing history. I would avoid repeating "remov{es/ing} commits in a dataset history" twice. And as stated above, I don't think the last sentence should be there because I don't think we should have a separate `drop` command.